content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: SelctPanel
+reactId: selectpanel
+figma:
+description: SelectPanel is a semantic dialog that allows for complex selection options within an overlay.
```

Comment:
```suggestion
description: Select panel is a semantic dialog that allows for complex selection options within an overlay.
```

This is just a grammatical formatting decision I think we established at one point, so just suggesting changes to be consistent with that (not trying to be nitpicky!)